src/main/java/com/exasol/adapter/metadata/DataType.java

```diff
@@ -54,6 +43,16 @@ public static DataType createVarChar(final int size, final ExaCharset charset) {
         return type;
     }
 
+    /**
+     * Create a <code>VARCHAR</code> data type with the maximum size supported by Exasol
+     *
+     * @param charset character set to be used when creating the data type
+     * @return data type
```

Comment:
return `varchar` data type.

same for others below.

Reply:
Done.